src/fsharp/FSComp.txt

```diff
@@ -1328,3 +1328,4 @@ tcTupleStructMismatch,"One tuple type is a struct tuple, the other is a referenc
 3210,tcNamedActivePattern,"%s is an active pattern and cannot be treated as a discriminated union case with named fields."
 3211,DefaultParameterValueNotAppropriateForArgument,"The default value does not have the same type as the argument. The DefaultParameterValue attribute and any Optional attribute will be ignored. Note: 'null' needs to be annotated with the correct type, e.g. 'DefaultParameterValue(null:obj)'."
 tcGlobalsSystemTypeNotFound,"The system type '%s' was required but no referenced system DLL contained this type"
+3213,parsUnexpectedSymbolEqualsInsteadOfIn,"Unexpected symbol '=' in expression, were you intending to use 'for x in y .. z do' instead?"
```

Comment:
Please use two sentences, along the lines "This is the error.  This is the analysis. This is the advice." thanks